taichi/backends/opengl/aot_data.h

```diff
@@ -44,8 +30,8 @@ struct CompiledFieldData {
 };
 
 struct AotData {
-  std::vector<AotCompiledKernel> kernels;
-  std::vector<AotCompiledKernelTmpl> kernel_tmpls;
+  std::unordered_map<std::string, CompiledProgram> kernels;
```

Comment:
Shall we rename this to `CompiledTaichiKernel`?

Reply:
Agreed! Will do a followup PR for that!